### PR TITLE
Added codecov configuration

### PR DESCRIPTION
**Pull request by fatroom**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
+[![Build Status](https://travis-ci.org/zalando-incubator/gson-money-typeadapter.svg?branch=master)](https://travis-ci.org/zalando-incubator/gson-money-typeadapter) [![codecov](https://codecov.io/gh/zalando-incubator/gson-money-typeadapter/branch/master/graph/badge.svg)](https://codecov.io/gh/zalando-incubator/gson-money-typeadapter)
```
Comment (twz123) [inline README.md]: Can you put each badge on its own line? Looks a bit more structured like that.

File: pom.xml [hunk 2]
```diff
@@ -69,11 +69,34 @@
...
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.7.7.201606060606</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>prepare-agent</goal>
+                        </goals>
+                    </execution>
+                    <execution>
+                        <id>report</id>
```
Comment (twz123) [inline pom.xml]: `jacoco-report`, or no ID at all?

File: src/test/java/org/zalando/gson/money/MonetaryAmountAdapterTest.java [hunk 3]
```diff
@@ -146,7 +145,7 @@ public void deserializeMonetaryAmountCollection() throws Exception {
                 "[null, {\"amount\":15.78,\"currency\":\"EUR\"}]", token.getType());
 
         assertThat(amountCollection, hasSize(2));
-        assertThat(amountCollection, contains(nullValue(), equalTo(buildMoney(15.78, "EUR"))));
+        assertThat(amountCollection, hasItems(buildMoney(15.78, "EUR"), null));
```
Comment (twz123) [inline src/test/java/org/zalando/gson/money/MonetaryAmountAdapterTest.java]: These changes are unrelated to the PR topic.

Comment (fatroom) [inline src/test/java/org/zalando/gson/money/MonetaryAmountAdapterTest.java]: Generally it's for fixing compiler warnings, but yeah. will cleanup

File: pom.xml (hunk 2 above)
Comment (fatroom) [inline pom.xml]: Id is mandatory. Switched to jacoco-report

File: src/test/java/org/zalando/gson/money/MonetaryAmountAdapterTest.java (hunk 3 above)
Comment (fatroom) [inline src/test/java/org/zalando/gson/money/MonetaryAmountAdapterTest.java]: Not there anymore

**Comment by twz123** [comment]
:+1:

**Comment by twz123** [comment]
What's wrong with Travis? The builds aren't starting…

**Comment by fatroom** [comment]
👍
